STUDIO/org.openl.rules.webstudio/webapp/pages/modules/administration/settings/mail/mailConfig.xhtml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui:composition xmlns:ui="http://java.sun.com/jsf/facelets">
+
+    <style>
+        #mail-config-verification-section {
+            margin-top: 0;
+            display:none
+        }
+    </style>
+    <div class="fields">
+        <section>
+            <h3>Mail configuration</h3>
+            <div>
+                <table>

```

Comment:
Don't use table elements. Replace it with div: https://www.w3.org/TR/css-flexbox-1/